### PR TITLE
Loadtesting

### PR DESCRIPTION
**Pull request by ridingwolf**

### CODE REVIEW

**Comment by ridingwolf** [comment]
todo:
- add the filtered lists for all registries
- add addressmatch
- review the distribution of calls per registry 
- review the distribution of calls to registries (weight set in `ApiSimulation.scala`)

File: user-files/simulations/infrastructure/RegistrySimulation.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.core.structure.PopulationBuilder
+
+import scala.concurrent.duration._
+import scala.math.{min, max}
+
+abstract class RegistrySimulation extends Simulation {
+  
+  private val loadTestApiKey = "REPLACEME"
```
Comment (CumpsD) [inline user-files/simulations/infrastructure/RegistrySimulation.scala:13]: Get this from vars

File: user-files/simulations/infrastructure/RegistrySimulation.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  val httpProtocol = http
+  // val httpProtocol = httpConfigFlood
+    .baseUrl("https://api.basisregisters.vlaanderen/v1")
```
Comment (CumpsD) [inline user-files/simulations/infrastructure/RegistrySimulation.scala:17]: Get this from vars

File: user-files/simulations/infrastructure/RegistrySimulation.scala [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    .header("x-api-key", loadTestApiKey)
+    .warmUp("https://www.vlaanderen.be/nl")
```
Comment (CumpsD) [inline user-files/simulations/infrastructure/RegistrySimulation.scala:19]: Get this from vars. `https://api.basisregisters.vlaanderen/v1/versions`

**Review by CumpsD — CHANGES_REQUESTED** [review]
Place some stuff in vars please

**Pull request merged**